src/test/resources/outputs/partiql/basics/select.sql

```diff
@@ -2,7 +2,9 @@
 SELECT "T"['a'] AS "a", "T"['b'] AS "b", "T"['c'] AS "c" FROM "default"."T" AS "T";
 
 --#[select-01]
-SELECT "T".* FROM "default"."T" AS "T";
+SELECT "T"['a'] AS "a", "T"['b'] AS "b", "T"['c'] AS "c", "T"['d'] AS "d", "T"['x'] AS "x", "T"['array'] AS "array", "T"['z'] AS "z", "T"['v'] AS "v", "T"['timestamp_1'] AS "timestamp_1", "T"['timestamp_2'] AS "timestamp_2" FROM "default"."T" AS "T";
```

Comment:
self-review: due to the way `SELECT *` is proactively expanded in the PLK-1.x planner, it's harder to reconstruct the `*`. This is one notable difference from the previous Scribe version. We can support reconstructing the `*` if there's a use case in the future, but it will require some changes in the PLK planner.

Reply:
I think we ought to open an issue to revisit this. It could be quite surprising to transpile a `Select *` query and get out something radically larger.

Reply:
Cut an issue to track -- https://github.com/partiql/partiql-scribe/issues/82. It would likely need some modifications to PLK to make the unplanning of unexpanded SELECT * more straightforward.